local-tests/chainrunner/node.py

```diff
@@ -28,9 +28,15 @@ def __init__(self, binary, chainspec, path, logdir=None):
     def _stdargs(self):
         return ['--base-path', self.path, '--chain', self.chainspec]
 
-    def start(self, name):
-        """Start the node. `name` is used to name of the logfile and for --name flag."""
-        cmd = [self.binary, '--name', name] + self._stdargs() + flags_from_dict(self.flags)
+    def _nodeargs(self, backup=True):
+        res = ['--node-key-file', op.join(self.path, 'p2p_secret'), '--enable-log-reloading']
+        if backup:
```

Comment:
I guess that we are missing: `else: res += ['--no_backup']` (one of these flags is mandatory, iirc)

Reply:
Nope. The idea behind this `backup` switch is to be able to run also older binaries, where the backup-path/no-backup was not yet a thing.

Reply:
so for newer binaries we always use backup, right?

Reply:
Yes, for now `--no-backup` is not supported